esp/smc/SMCLib/TpContainer.cpp

```diff
@@ -743,11 +743,36 @@ static void refreshValidTargets()
     }
 }
 
+static void addUniqueDataPlaneName(const char * dataPlaneName)
+{
+    if (isEmptyString(dataPlaneName))
+        return;
+
+    if (validDataPlaneNames.find(dataPlaneName) == validDataPlaneNames.end())
+    {
+        validDataPlaneNames.insert(dataPlaneName);
+        PROGLOG("adding valid data plane name: %s", dataPlaneName);
```

Comment:
This could be a bit too verbose.

Reply:
How about changing to: 'adding data plane: %s'?

Reply:
It was more the number of lines than the text.  @afishbeck do you mind?

Reply:
I'd rather not have the excessive logging.  If it ends up merged this way could streamline later.